<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MAA011253A" failures="0" tests="3" name="com.raj.tc.TC001_RAJBasicDetails" time="391.909" errors="1" timestamp="2020-11-25T18:09:25 IST" skipped="0">
  <testcase classname="com.raj.tc.TC001_RAJBasicDetails" name="basicDetails" time="195.872"/>
  <testcase classname="com.raj.tc.TC001_RAJBasicDetails" name="afterClass" time="0.789">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at lib.utils.HTMLReporter.reportStep(HTMLReporter.java:73)
at lib.selenium.WebDriverServiceImpl.closeAllBrowsers(WebDriverServiceImpl.java:495)
at lib.selenium.PreAndPost.afterClass(PreAndPost.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- afterClass -->
  <testcase classname="com.raj.tc.TC001_RAJBasicDetails" name="basicDetails" time="195.248"/>
</testsuite> <!-- com.raj.tc.TC001_RAJBasicDetails -->
